### PR TITLE
[DOCS] Updating embedded android docs

### PR DESCRIPTION
**Pull request by rops**
Updating outdated Android embedded app. Fix _https://github.com/facebook/react-native/issues/6422_

( First contribution 🎉 )

### CODE REVIEW

File: docs/EmbeddedAppAndroid.md [hunk 1]
```diff
@@ -166,7 +129,3 @@ To run your app, you need to first start the development server. To do this, sim
 Now build and run your Android app as normal (e.g. `./gradlew installDebug`). Once you reach your React-powered activity inside the app, it should load the JavaScript code from the development server and display:
 
 ![Screenshot](img/EmbeddedAppAndroid.png)
-
-## Sharing a ReactInstance across multiple Activities / Fragments in your app
```
Comment (rops) [inline docs/EmbeddedAppAndroid.md]: not sure if this is still supported actually

Comment (satya164) [inline docs/EmbeddedAppAndroid.md]: There have been no changes regarding this. So it's still possible.

Comment (rops) [inline docs/EmbeddedAppAndroid.md]: alright. Reverting this then

**Comment by mkonicek** [comment]
I think @foghina wrote the original guide :)

File: docs/EmbeddedAppAndroid.md (hunk 1 above)
Comment (foghina) [inline docs/EmbeddedAppAndroid.md]: Yes please revert this. It's not super up-to-date but I think it's worth leaving in as a starting point.

**Comment by foghina** [comment]
Looks good. I'd put the repository config first and then adding the dependency. But I only feel strongly about leaving the singleton holder stuff in, revert that and we can merge.

**Comment by facebook-github-bot-8** [comment]
@rops do you have any updates for this pull request? It's been a while so we wanted to check in and see if you've looked at the requested changes.

**Comment by mkonicek** [comment]
@rops I'm going through all pull requests and noticed this one hasn't been updated in a while and the last comment requests changes. I'll close this pull request so it doesn't stay open indefinitely but please send a new one if you want to continue working on this.

**Pull request closed without merging**